src/components/map-footer/map-footer.js

```diff
@@ -0,0 +1,44 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import styles from './map-footer-styles.scss';
+
+class MapFooter extends PureComponent {
```

Comment:
The MapFooter is different in the new designs:
![image](https://user-images.githubusercontent.com/9701591/48563132-ea46b700-e8f3-11e8-9c76-e28fd9cb59df.png)